internal/backend/crypto/age/ssh.go

```diff
@@ -69,6 +66,24 @@ func (a *Age) getSSHIdentities(ctx context.Context) (map[string]age.Identity, er
 	return ids, nil
 }
 
+func getSSHDir() (string, error) {
+	preferredPath := os.Getenv("GOPASS_SSH_DIR")
+	sshDir := filepath.Join(preferredPath, ".ssh")
+	if fsutil.IsDir(sshDir) && sshDir != "" {
```

Comment:
This should actually check if prefrerredPath is empty. Otherwise we might still pick up a .ssh dir does exist in the current pwd.